docs/hooks/useUpdate.mdx

```diff
@@ -0,0 +1,65 @@
+---
+name: useUpdate
+menu: Hooks
+---
+
+import { useState } from 'react';
+import { Playground } from 'docz';
+import { useToggle, useUpdate } from '../../src';
+import PropsTable from '../utils/PropsTable';
+
+# useUpdate
+
+## Purpose
+
+A replacement for componentDidUpdate in React. It will also be executed on first render.
+
+## Usage
+
+Effect useUpdate will be called only on count state update
+
+<Playground>
+  {() => {
+    const [count, setCount] = useState(0);
+
+    const styles = {
+      container: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'center',
+        backgroundColor: count % 2 === 0 ? '#81a1c1' : '#d8dee9',
+        padding: 20,
+      },
+      button: {
+        backgroundColor: count % 2 === 0 ? 'tomato' : 'gray',
+        color: 'white',
+        border: 0,
+        padding: '10px 30px',
+        margin: '0 10px',
+        outline: 0,
+        cursor: 'pointer'
+      },
+    }
+
+    useUpdate({
+    action: () => console.log('Count was updated'),
+    exit: () => console.log('I am no longer being used'),
+
+}, count)
```

Comment:
Can you kindly indent the code in .mdx file, running prettier only ensures that the maxLineLength & no-white-spaces check is maintained.

Reply:
Do ensure that 2 tabs spaces indentation style for code is maintained in .mdx file.